The-Chonky-Panda/Atelier
# Submit review refactor

## Description

Split `reviewModal` into smaller components, for better readability and testing.
Added validation style to mandatory fields.
Removed `http://127.0.0.1:3000` to make fetches work outside local development.

## How to test

No major functionality change - just verify that with invalid inputs there are a style change

## Notes

Mainly want to get this in because our fetch calls are failing on AWS, so want to push up a fix sooner than later

## Issue tracking

https://trello.com/c/RpIiO4L6
https://trello.com/c/tNkYviaE
https://trello.com/c/492d8RV2
